clients/src/main/java/org/apache/kafka/clients/admin/internals/ListConsumerGroupOffsetsHandler.java

```diff
@@ -73,7 +73,10 @@ public String apiName() {
     public OffsetFetchRequest.Builder buildRequest(int coordinatorId, Set<CoordinatorKey> keys) {
         // Set the flag to false as for admin client request,
         // we don't need to wait for any pending offset state to clear.
-        return new OffsetFetchRequest.Builder(groupId.idValue, false, partitions, false);
```

Comment:
The old `Builder` still exists, I just changed it to use the new format, since when we implement the Admin APIs we will have to change everything to use that anyway. However if you think it may be better to retain the old pattern I can change it - what do you think?

Reply:
We can leave the admin client changes for the next PR to keep the changes in this one small for 3.0.

Reply:
Okay, I have removed it.